### PR TITLE
ci(stale): add scheduled workflow

### PR DESCRIPTION
**Pull request by morey-tech**
On weekdays, this workflow will run once in the morning*. If an issue of `priority/low` or `priority/wontfix` has been inactive for 90 days, it will be marked as stale (by adding the `stale` label), and a comment will be left. If it remains inactive for another 30 days, it will be closed (but can be re-opened if needed, i.e. not locked).

### CODE REVIEW

File: .github/workflows/stale.yml
Comment (morey-tech) [inline .github/workflows/stale.yml:30]: I went a bit aggressive on the close/stale timings. I wonder for issues if we should exempt ones that have an `accepted` (or some other verb) so that they won't be considered stale

Comment (krancour) [reply .github/workflows/stale.yml:30]: That's a really good idea.

I can think of a number of existing labels that would be a good indicator that we are committed to working on something, but if we want to start with the easiest thing and tweak it as necessary, what if we exempt anything labeled `priority/normal|high|urgent`.

If we do a decent job triaging issues, all of those probably reflect that at least one maintainer thinks the issue needs to be addressed.

File: .github/workflows/stale.yml
Comment (krancour) [inline .github/workflows/stale.yml:17]:
```suggestion
          stale-issue-label: 'stale'
```

Comment (morey-tech) [reply .github/workflows/stale.yml]: Good catch, fixed in https://github.com/akuity/kargo/pull/908/commits/7505c22e5eca20ba86829b3f85fb5608a94fb6e2

File: .github/workflows/stale.yml
Comment (morey-tech) [reply .github/workflows/stale.yml:30]: That's an elegant solution; love it. Added in https://github.com/akuity/kargo/pull/908/commits/7505c22e5eca20ba86829b3f85fb5608a94fb6e2

File: .github/workflows/stale.yml
Comment (morey-tech) [inline .github/workflows/stale.yml:33]: `on hold` or `on-hold`? Or push people to use drafts instead of a label for PRs that are not ready?

Comment (krancour) [reply .github/workflows/stale.yml:33]: Draft is probably good enough for now. We'll learn what's not working and tweak it later.

**Comment by morey-tech** [comment]
Updated to only target issues, after 90 days.

File: .github/workflows/stale.yml
Comment (krancour) [inline .github/workflows/stale.yml:18]: Nit: Should this mention the 90/30?

LGTM either wait.

**Review by krancour — APPROVED** [review]

File: .github/workflows/stale.yml
Comment (morey-tech) [reply .github/workflows/stale.yml:18]:
```suggestion
            This issue has been automatically marked as stale because it had no activity for 90 days. It will be closed if no activity occurs in the next 7 days but can be reopened if it becomes relevant again.
```

**Pull request merged**
